### PR TITLE
[TA] Added displayName property

### PR DESCRIPTION
**Pull request by mssfang**
- Fixed live test failure at `healthcareLroWithOptions` test
- Added `displayName` property to the following classes to set the optional display name:
    - `AnalyzeHealthcareEntitiesOptions`
    - `MultiLabelClassifyOptions`
    - `RecognizeCustomEntitiesOptions`
    - `SingleLabelClassifyOptions`
- Added `displayName` property to the following operations to read the optional display name set on options classes above:
    - `AnalyzeHealthcareEntitiesOperationDetail` from `AnalyzeHealthcareEntitiesOptions`
    - `ClassifyDocumentOperationDetail` from `MultiLabelClassifyOptions` and `SingleLabelClassifyOptions`
    - `RecognizeCustomEntitiesOperationDetail` from `RecognizeCustomEntitiesOptions`

### CODE REVIEW

File: sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,15 @@
 ## 5.2.0-beta.5 (Unreleased)
 
 ### Features Added
+- Added `displayName` property to the following classes to set the optional display name:
```
Comment (kristapratico) [inline sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md:6]: maybe specify that it is the display name for the job/operation?

File: sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TestUtils.java [hunk 2]
```diff
@@ -899,24 +899,15 @@ static AnalyzeHealthcareEntitiesResult getRecognizeHealthcareEntitiesResult2() {
         HealthcareEntityPropertiesHelper.setDataSources(healthcareEntity2,
             IterableStream.of(Collections.emptyList()));
         final HealthcareEntity healthcareEntity3 = new HealthcareEntity();
-        HealthcareEntityPropertiesHelper.setText(healthcareEntity3, "ST depressions");
-        HealthcareEntityPropertiesHelper.setNormalizedText(healthcareEntity3, "ST segment depression (finding)");
+        HealthcareEntityPropertiesHelper.setText(healthcareEntity3, "ST depressions in the anterior lateral leads");
```
Comment (kristapratico) [inline sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TestUtils.java:902]: interesting - are these updates due to changes in the service?

**Review by kristapratico — APPROVED** [review]

File: sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TestUtils.java (hunk 2 above)
Comment (mssfang) [reply sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TestUtils.java:902]: Yes. I ran the live test to record the response but found it is no longer a valid test.

File: sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md (hunk 1 above)
Comment (heaths) [reply sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md:6]: Looks like he should've copied my text after I made the change based on your same feedback on my PR. 😀

File: sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClientTest.java [hunk 3]
```diff
@@ -2566,9 +2579,11 @@ public void multiLabelClassification(HttpClient httpClient, TextAnalyticsService
         client = getTextAnalyticsAsyncClient(httpClient, serviceVersion);
         classifyCustomMultiLabelRunner((documents, parameters) -> {
             SyncPoller<ClassifyDocumentOperationDetail, ClassifyDocumentPagedFlux> syncPoller =
-                client.beginMultiLabelClassify(documents, parameters.get(0), parameters.get(1), "en", null).getSyncPoller();
+                client.beginMultiLabelClassify(documents, parameters.get(0), parameters.get(1), "en",
+                    new MultiLabelClassifyOptions().setDisplayName("operationName")).getSyncPoller();
             syncPoller = setPollInterval(syncPoller);
-            syncPoller.waitForCompletion();
+            PollResponse<ClassifyDocumentOperationDetail> pollResponse = syncPoller.waitForCompletion();
+            assertEquals("operationName", pollResponse.getValue().getDisplayName());
```
Comment (deyaaeldeen) [inline sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClientTest.java:2586]: nitpick: The test case can be more robust if the "operationName" value comes directly from the options bag object, so if for some reason it changed, there is no need to change the assert line too.

**Review by deyaaeldeen — APPROVED** [review]

**Pull request merged**